.github/workflows/build.yml

```
@@ -96,13 +102,7 @@ jobs:
           for TAG in "${BUILD_TAGS[@]}"; do
               echo "${TAG}"
           done
-          echo "alias_tags=${alias_tags[*]}" >> $GITHUB_OUTPUT
-
-      - name: Get current version
-        id: labels
-        run: |
-          ver=$(skopeo inspect docker://${{ env.BASE_IMAGE_URL }}:${{ env.FEDORA_MAJOR_VERSION }} | jq -r '.Labels["org.opencontainers.image.version"]')
-          echo "VERSION=$ver" >> $GITHUB_OUTPUT
+          echo "alias_tags=${alias_tags[*]}" >> $GITHU:B_OUTPUT
```

```suggestion
          echo "alias_tags=${alias_tags[*]}" >> $GITHUB_OUTPUT
```
this may be your issue?

---

thanks, totally missed that since that wasn't an intentional change and the GH diff didn't really show it that well

---

yeah, that was it